src/test/java/seedu/address/logic/commands/DeletePersonFromModuleCommandIntegrationTest.java

```diff
@@ -0,0 +1,166 @@
+import seedu.address.model.module.ModuleCode;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.testutil.ModuleBuilder;
+import seedu.address.testutil.TypicalModules;
+import seedu.address.testutil.TypicalPersons;
+
+/**
+ * Contains integration tests (interaction with the Model) for {@code DeletePersonFromModuleCommand}.
+ */
+public class DeletePersonFromModuleCommandIntegrationTest {
+
+    private Model model;
+
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(getTypicalAddressBookWithAssociations(), new UserPrefs());
+    }
+
+    @Test
+    public void execute_deletePersonFromModuleAtHome_success() {
+        Module validModuleWithAliceAndBenson = TypicalModules.CS2106_WITH_ALICE_BENSON;
+
+        ModuleCode validModuleCode = validModuleWithAliceAndBenson.getModuleCode();
+        Name aliceName = ALICE.getName();
+
+        model.goToHomePage();
+
+        Set<Person> expectedPersons = new HashSet<>(Arrays.asList(BENSON));
+        Module expectedModule = new ModuleBuilder(validModuleWithAliceAndBenson).withPersons(expectedPersons).build();
+
+        Model expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+        expectedModel.goToHomePage();
+        expectedModel.setModule(validModuleWithAliceAndBenson, expectedModule);
+
+        assertCommandSuccess(new DeletePersonFromModuleCommand(validModuleCode, aliceName), model,
+                String.format(DeletePersonFromModuleCommand.MESSAGE_DELETE_PERSON_FROM_MODULE_SUCCESS,
+                        validModuleCode, aliceName),
+                expectedModel);
+    }
+
+    @Test
+    public void execute_deletePersonFromModuleNotAtHome_success() {
+        Module validModuleWithAliceAndBenson = TypicalModules.CS2106_WITH_ALICE_BENSON;
+
+        ModuleCode validModuleCode = validModuleWithAliceAndBenson.getModuleCode();
+        Name aliceName = ALICE.getName();
+
+        Predicate<Module> modulePredicate = module -> module.equals(validModuleWithAliceAndBenson);
+        Predicate<Person> personPredicate = person -> person.equals(ALICE) || person.equals(BENSON);
+
+        // Simulate not at home.
+        model.updateFilteredModuleList(modulePredicate);
+        model.updateFilteredPersonList(personPredicate);
+        model.setHomeStatus(false);
+
+        Set<Person> expectedPersons = new HashSet<>(Arrays.asList(BENSON));
+        Module expectedModule = new ModuleBuilder(validModuleWithAliceAndBenson).withPersons(expectedPersons).build();
+
+        Model expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+
+        // Simulate not at home.
+        expectedModel.updateFilteredModuleList(modulePredicate);
+        expectedModel.updateFilteredPersonList(personPredicate);
+        expectedModel.setHomeStatus(false);
+
+        // After setting new module, the person list is updated to be now without the deleted person.
```

Comment:
```suggestion
        // After setting new module, the person list is now updated to be without the deleted person.
```
Might this phrasing be clearer?

Reply:
Yes, I think it does! I shall adopt the necessary changes 🛩️